packages/core/src/libraries/sign-in-experience/index.ts

```diff
@@ -72,26 +72,27 @@ export const createSignInExperienceLibrary = (
   const getSignInExperienceForApplication = async (
     applicationId?: string
   ): Promise<SignInExperience & { notification?: string }> => {
-    const signInExperience = await findDefaultSignInExperience();
-
     // Hard code Admin Console sign-in methods settings.
     if (applicationId === adminConsoleApplicationId) {
       return {
-        // If we need to hard code, it implies Logto is running in the single-tenant mode;
-        // Thus we can hard code Tenant ID as well.
-        tenantId: defaultTenantId,
         ...adminConsoleSignInExperience,
+        tenantId: adminTenantId,
         branding: {
           ...adminConsoleSignInExperience.branding,
           slogan: i18next.t('admin_console.welcome.title'),
         },
-        termsOfUseUrl: signInExperience.termsOfUseUrl,
-        languageInfo: signInExperience.languageInfo,
+        termsOfUseUrl: null,
+        languageInfo: {
+          autoDetect: true,
+          fallbackLanguage: 'en',
+        },
```

Comment:
We should not hard code the Admin Console SIE settings, because we should able to config the sigin in methods.
<img width="568" alt="image" src="https://user-images.githubusercontent.com/10806653/218607748-d5576a19-0262-4a9f-bd75-8f4a8540005c.png">

Reply:
Hard-coded SIE for admin console should be fine for now, I think, and we can always change the hard-coded value to configure it at a later time

Reply:
yes i'll do this change later

Reply:
added LOG-5402 LOG-5407